protocol/slp/slp.md

```diff
@@ -0,0 +1,78 @@
+# Simple Ledger Protocol
+
+Simple Ledger Protocol is a token system for the Bitcoin Cash network.
+It allows users to create, issue, and transfer digital tokens that enjoy the same security model and network of Bitcoin Cash.
+Users can associate the created tokens with assets and values, and thus, utilize the [Blockchain](/protocol/blockchain) as the public ledger to achieve transparency and integrity for their transactions.
+
+
+## The intent of token systems
+
+The Blockchain provides an open ledger with transparency and integrity to document transactions.
+However, to utilize this ledger, one has to devise rules and security models for the transactions, as well as the computational power to support a network that complies with the rules.
+The above costs make such applications prohibitively expensive for most individual users, while also hinders the emergence of coins for specific applications and transactions.
+Such needs drive the creation of various token systems on Bitcoin Cash.
+Token systems are additional layers of protocols on Bitcoin Cash, that allow the creation, issuance, and transfer of tokens on the Bitcoin Cash network.
+Users and creators of the tokens can utilize the computational power and other benefits of the Bitcoin Cash network for their own token transactions.
+
+
+## The design of Simple Ledger Protocol
+
+Simple Ledger Protocol (SLP) is one of the most prevalent token systems on Bitcoin Cash.
+SLP employs a “colored coins” design that associates token amounts with BCH [transaction](/protocol/blockchain/transaction) outputs.
+The first output of an SLP transaction will utilize the OP_RETURN to return messages with the predefined format to annotate the amount of SLP tokens associated with each transaction output in the same transaction.
```

Comment:
I placed the link in the Consensus Rules section since I want this paragraph to be a high-level overview of the protocol, while the Consensus Rules section serves as the accurate description of the protocol.

Reply:
If that's the case then I think maybe you're getting too detailed.  If you're mentioning `OP_RETURN` then I think we should link to the content for that type of output.  It probably even just makes sense to call it a data output and link it.  But I would also be fine with just leaving out how it does it for now and just generically mentioning that the transaction will include data that specifies the amounts for the outputs.

Reply:
My original thought was to provide a brief summary for people of different backgrounds and purposes. Therefore, I included some iconic technical details about SLP (message types, utilization of OP_RETURN), while also tried to limit their footprints so non-technical readers can still get the rough idea. However, in hindsight, I do realize that the particular language issue regarding OP_RETURN may indeed cause more confusion than providing information. I think calling it a data output here is the proper decision here.